src/Assets/pageTitleTags.ts

```diff
@@ -0,0 +1,17 @@
+const pageTitleTags: Record<string, string> = {
+  'step-1': 'Preferred Language',
```

Comment:
Do we want to do `My Friend Ben | Preferred Language`.

Reply:
I think that just having the `Preferred Language` is fine because otherwise the `My Friend Ben | Preferred Language` string just runs off the tab:
<img width="171" alt="Screenshot 2023-11-09 at 11 44 57 AM" src="https://github.com/Gary-Community-Ventures/benefits-calculator/assets/86989161/fadad8bf-00e5-492d-ae04-0fce3169573b">